### ISSUE TITLE
zjstatus plugin does not appear in new sesssion until after session-serialization interval

### ISSUE THREAD

**Issue by mike-lloyd03** [zellij-org/zellij#2937]
On 0.39.1, zjstatus plugin does not always immediately display when a new session starts. Probably about half the time it appears immediately. If it does not appear immediately, it will appear after whatever time is configured for the session-serialization interval.

**Basic information**

`zellij --version`: zellij 0.39.1
`stty size`: 91 242
`uname -av` or `ver`(Windows): Linux dev 6.5.6-arch2-1 #1 SMP PREEMPT_DYNAMIC Sat, 07 Oct 2023 08:14:55 +0000 x86_64 GNU/Linux

**Zellij Log**
```
ERROR  |zellij_server::plugins   | 2023-11-13 12:28:00.423 [wasm      ] [zellij-server/src/plugins/mod.rs:432]: Plugin with id: 0 not found
INFO   |zellij_server::plugins::p| 2023-11-13 12:28:00.448 [async-std/runti] [zellij-server/src/plugins/plugin_loader.rs:495]: Loaded plugin '/home/mike/.config/zellij/plugins/zjstatus.wasm' from cache folder at '/home/mike/.cache/zellij' in 23.677042ms
INFO   |zellij_server            | 2023-11-13 12:28:00.449 [async-std/runti] [zellij-server/src/lib.rs:919]: Compiling plugins using Cranelift
ERROR  |zellij_server::background| 2023-11-13 12:28:11.605 [async-std/runti] [zellij-server/src/background_jobs.rs:424]: Failed to read created stamp of resurrection file: Os { code: 2, kind: NotFound, message: "No such file or directory" }
```

The last line of the log entry repeats multiple times.

I've also opened [a ticket with zjstatus](https://github.com/dj95/zjstatus/issues/24)

**Comment by dj95** [comment]
This issue can be closed. It occurred because I tried to implement some kind of suspend behaviour, which does not render the plugin, when the client is detached. However, with the latest release, the session information I used with the information how many clients are connected to the sessions get's updated too late (in the next cycle of the session serialisation I guess), which suspended rendering the plugin.
